src/completion-provider.js

```diff
@@ -9,15 +9,27 @@ const internalModules = require('./internal-modules');
 
 class CompletionProvider {
   constructor() {
-    this.selector = '.source.js .string.quoted, .source.coffee .string.quoted';
+    this.selector = '.source.js .string.quoted, .source.coffee .string.quoted, .source.css .meta.property-value .string.quoted';
     this.disableForSelector = '.source.js .comment, source.js .keyword';
     this.inclusionPriority = 1;
   }
 
-  getSuggestions({editor, bufferPosition, prefix}) {
+  getSuggestions({editor, bufferPosition, prefix, scopeDescriptor}) {
     const line = editor.getTextInRange([[bufferPosition.row, 0], bufferPosition]);
-    if (!/require|import/.test(line)) {
-      return [];
+
+    const sourceLanguage = scopeDescriptor.scopes[0]
+    switch (sourceLanguage) {
```

Comment:
Please do not copy-paste.
Add function `getTestRegExp(sourceLanguage)`, for example.